### DISCUSSION TITLE
How to change the animation?

### DISCUSSION THREAD

**Discussion by Sulphy77** [Schneegans/Burn-My-Windows#34] [Q&A] [answered]
Probably a dumb que..... i have this working on Ubuntu, but how the heck do i change the animation to Matrix?
Much appreciated, this is very cool!!

**Comment by Schneegans** [comment, marked as answer]
The _Matrix_ and _T-Rex-Attack_ shaders are only supported since GNOME 40 (due to technical limitations of GNOME 3.3x). If you are on GNOME 40 (Ubuntu 21.10), then you just have to open the extension's settings and choose the desired animation from the drop down menu.

**Reply by Sulphy77** [reply]
Thanks for that, I am indeed on Ubuntu 21.10, i dont see burn my windows in the list of applications in extensions.... 
Apologies, i am not that great at Linux!! So, it wouldnt surprise me if you told me i missed something! 
Appreciate your help and patience!
Maybe if you had a screenshot to share?

**Reply by Schneegans** [reply]
Well, there is actually not much which could be done wrong :wink: 
1. Go to https://extensions.gnome.org/extension/4679/burn-my-windows/ and flip the switch.
2. Go to https://extensions.gnome.org/local/ and click the settings icon next to the extension.

Does this work for you?

**Reply by Sulphy77** [reply]
AH.... okay, so it wasnt working at all.... then i came across the issue with the SNAP version of FF, and not the deb version! I had to uninstall the snap version, and install the other version... got it working now! Thanks so much for your help!
FYI, this is what i followed : https://askubuntu.com/questions/1369911/gnome-browser-addon-for-installing-extensions-does-not-work-on-firefox-in-ubuntu
The workaround marked as answer did the trick for me!

**Reply by Schneegans** [reply]
Ahh, good to hear!
